go-controller/pkg/ovn/acl.go

```diff
@@ -2,12 +2,86 @@ package ovn
 
 import (
 	"fmt"
+	"strings"
+
 	libovsdbclient "github.com/ovn-org/libovsdb/client"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+
+	knet "k8s.io/api/networking/v1"
+)
+
+func joinACLName(substrings ...string) string {
+	return strings.Join(substrings, "_")
+}
+
+// aclType defines when ACLs will be applied (direction and pipeline stage).
+// All acls of the same type will be sorted by priority, priorities for different types are independent.
+type aclType string
+
+const (
+	// lportIngress will be converted to direction="to-lport" ACL
+	lportIngress aclType = "Ingress"
+	// lportEgressAfterLB will be coverted to direction="from-lport", options={"apply-after-lb": "true"} ACL
+	lportEgressAfterLB aclType = "Egress"
```

Comment:
Be aware that if this gets printed in a log we will see `Egress` which might not be conveying the same amount of information as the constant name is.